components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/edit/EditToolbar.kt

```diff
@@ -99,7 +99,7 @@ internal class EditToolbar(
         scaleType = ImageView.ScaleType.CENTER
 
         setOnClickListener {
-            urlView.text.clear()
+            urlView.setText("")
```

Comment:
Oh, I remember we fixed such a thing in Focus too. It would be super helpful to add a comment here explaining why we use `setText("")` here. Otherwise someone will come along and change it to clear() again. :)
You could even look into adding a test that makes sure `setText()` is called in this situation.